content/200-concepts/100-components/02-prisma-client/260-client-extensions/110-client.mdx

```diff
@@ -0,0 +1,70 @@
+---
+title: 'Add methods to Prisma Client'
+metaTitle: 'Prisma Client extensions: client (Preview)'
+metaDescription: 'Extend the functionality of Prisma Client'
+tocDepth: 3
+---
+
+<TopBlock>
+
+<Admonition type="info">
+
+This page is in draft.
+
+</Admonition>
+
+From version **TBD**, you can use `client` [Prisma Client extensions](/concepts/components/prisma-client/client-extensions) to add top-level methods to Prisma Client.
+
+To create Prisma Client extensions, use the [`$extends`](/concepts/components/prisma-client/client-extensions#about-prisma-client-extensions) client-level method. Use the `client` extension type to extend Prisma Client.
+
+</TopBlock>
+
+## Enable the preview feature
+
+Before you create Prisma Client extensions, you must enable the `clientExtensions` feature flag in the `generator` block of your `schema.prisma` file, as follows:
+
+```prisma
+generator client {
+  provider        = "prisma-client-js"
+  previewFeatures = ["clientExtensions"]
+}
+```
+
+## Extend Prisma Client
+
+To add methods to Prisma Client, use the following structure:
+
+```ts
+const xprisma = prisma.$extends({
+  client?: { ... }
+})
+```
+
+- `client`: An optional field that adds a top-level method to the client
+
+The following example uses a `client` extension to start an [interactive transaction](/concepts/components/prisma-client/transactions#interactive-transactions-in-preview) without callbacks.
```

Comment:
Does the example actually still show that? I think the example is more general `$method` now, and just this sentence and the usage example in line 59-70 still is about interactive transactions.

Reply:
D'oh! I think I had two examples in there and I removed part of one and part of the other. I've removed the `method` example now, and reinstated the code that is actually about an interactive transation